fbw-a32nx/src/systems/instruments/src/PFD/shared/PFDSimvarPublisher.tsx

```diff
@@ -295,6 +302,14 @@ export enum PFDVars {
     trueTrackRaw = 'L:A32NX_ADIRS_IR_1_TRUE_TRACK',
     slatPosLeft = 'L:A32NX_LEFT_SLATS_ANGLE',
     fm1NavDiscrete = 'L:A32NX_FM1_NAV_DISCRETE',
+    fm1EisDiscrete2 = 'L:A32NX_FM1_EIS_DISCRETE_WORD_2',
```

Comment:
**question:** can't we put those in `FMBusPublisher`?

Reply:
We don't appear to be using that in the PFD? We should do that as a separate refactor.